router/router.go

```diff
@@ -44,6 +44,18 @@ import (
 	"github.com/rs/cors"
 )
 
+type aliasBidderNameHelper struct{}
+
+type getAliasBidderNames interface {
```

Comment:
how about renaming to `getAliasBidderNames` to `aliasToParentBidderName` and `aliasBidderNameHelper` to `aliasToParentBidderNameGetter`